ARCHITECTURE.md

```diff
@@ -0,0 +1,44 @@
+# Architecture
+
+The PDC is built using the following technologies:
+
+| Component                                                         | Technology                                        |
+|-------------------------------------------------------------------|---------------------------------------------------|
+| [service](https://github.com/PhilanthropyDataCommons/service)     | TypeScript (NodeJs)                               |
+| [front end](https://github.com/PhilanthropyDataCommons/front-end) | TypeScript (React)                                |
+| Authentication                                                    | [Keycloak](https://github.com/keycloak/keycloak/) |
+| [SDK generation](https://github.com/PhilanthropyDataCommons/sdk)  | Swagger Codegen                                   |
+| Document storage                                                  | S3                                                |
+| Data storage                                                      | Postgres                                          |
```

Comment:
There's something a bit off about this table, trying to put my finger on it.

I think it under-describes what "service" is (which needs to be described as providing the API), and over-prioritizes the front end (which belongs at the bottom of this list, if on here at all).

IMO the order should be something like: service/API, authentication, data storage, document storage, SDK, front end. Not that the table needs to reflect the "stack order" or something, but… that feels more coherent.

Also I'd double-check the name of each technology and use its canonical name — e.g. `Node.js` and `PostgreSQL` — rather than any shorthand. And and and I wonder if "S3" should be "S3-compatible service" or something?

Reply:
This was a bit of stream of consciousness.

Reply:
yeah. well. The table said to me there's something a bit off about YOU!!!!!!!!!!

(good feedback; will update tomorrow, although I truly fear the prospect of editing a markdown table)

Reply:
> I truly fear the prospect of editing a markdown table

You are not wrong to be intimidated by this.